scripts/setup_project.py

```
@@ -0,0 +1,206 @@
+    Execute a shell command and report its success or failure.
+    
+    Parameters:
+        command (str): The shell command to execute.
+        description (str): A brief description of the command's purpose, used in status messages.
+    
+    Returns:
+        bool: True if the command executes successfully, False otherwise.
+    """
+    print(f"🔄 {description}...")
+    try:
+        result = subprocess.run(command, shell=True, check=True, capture_output=True, text=True)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)